offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/overview/projectcreation/CreateProjectViewModel.groovy

```diff
@@ -115,7 +120,20 @@ class CreateProjectViewModel {
     private void resetModel() {
         initFields()
     }
-
+    
+    private void validateSpaceName() {
+        try {
+            ProjectSpace space = new ProjectSpace(desiredSpaceName)
+            this.setResultingSpaceName(space.name)
+            this.setSpaceIsValid(true)
+            this.setSpaceNameValidationResult("Space name is valid.")
+        } catch (IllegalArgumentException e) {
+            this.setSpaceIsValid(false)
+            this.setSpaceNameValidationResult("${desiredSpaceName} is not a valid " +
+                    "space name. Spaces are only allowed to contain alphanumeric characters, underscore (_) and minus (-).")
```

Comment:
did you test it? I think the variable is called spaceNameIsValid()

at least it fails for me because of that

Reply:
```suggestion
            ProjectSpace space = new ProjectSpace(desiredSpaceName)
            this.setResultingSpaceName(space.name)
            this.setSpaceNameIsValid(true)
            this.setSpaceNameValidationResult("Space name is valid.")
        } catch (IllegalArgumentException e) {
            this.setSpaceNameIsValid(false)
            this.setSpaceNameValidationResult("${desiredSpaceName} is not a valid " +
                    "space name. Spaces are only allowed to contain alphanumeric characters, underscore (_) and minus (-).")
```